html/about.php

```diff
@@ -13,29 +13,27 @@
           <div class="col-lg-7" data-aos="fade-up" data-aos-delay="100">
             <img src="assets/img/about.jpg" class="img-fluid mb-4" alt="">
```

Comment:
```suggestion
            <img src="<?= IMG_ABOUT?>" class="img-fluid mb-4" alt="">
```

Y en donde sea que hayan imagenes